### PR TITLE
wtffffffffffffffffff

### PR DESCRIPTION
**Pull request by fukinbubba**

### CODE REVIEW

**Review by fukinbubba — COMMENTED** [review]
Sorry. Been having password issues with same username and passwords connected.to th we boxes I initially bought from.  Using a cheap tablet an iPhone 14 and borrowing a laptop all on a Verizon at home wifi box in the middle of nowhere.  Everything I went to sign into new app it would open up on other screen etc.  I was reverted to etherscan.io  from bloxpacks. All of my data was crammed I guess. I'm no coder and mad respect for you guys. Trying to just find a place to message In between shifts at work has been hectic and long.    Every app I opened was linked to same email and my message log was backed so far up everything I would verify it was verifying something from 1500 days ago.  Once again. Soo sorry I don't know what I'm doing at all

**Comment by fukinbubba** [comment]
I delete etherscan.io acc?

**Comment by fukinbubba** [comment]
Tell me what I have to do.  Don't know anything really

**Review by fukinbubba — COMMENTED** [review]
?

**Review by fukinbubba — COMMENTED** [review]
? What should I be doing

File: docs/cdk/architecture/cdk-pp-highlevel-arch.md [hunk 1]
```diff
@@ -1,62 +1,17 @@
-The figure below depicts a simplified, high level view of the CDK PP chain architecture, as well as the flow of transactions through the system.
+The figure below depicts a simplified, high-level view of the CDK chain architecture using pessimistic proofs and the flow of transactions through the system.
 
-![Figure: CDK PP Architecture](../../img/cdk/cdk-pp-architecture-001.png)
+![Figure: CDK Architecture](../../img/cdk/CDK-pessimistic-proof-chain.jpg)
```
Comment (fukinbubba) [inline docs/cdk/architecture/cdk-pp-highlevel-arch.md:4]: 0x3b5379dAFD6675Dblab420000046D13a76b3849C

File: docs/cdk/architecture/cdk-pp-highlevel-arch.md [hunk 2]
```diff
@@ -1,62 +1,17 @@
...
 
-## Transaction flow
```
Comment (fukinbubba) [inline docs/cdk/architecture/cdk-pp-highlevel-arch.md:5]: API keys 
Etherscan {VZFDUWB3YGQ1YCDKTCU1D6DD}
BSCSCAN {ZM8ACMJB67C2IXKKBF8URFUNSY}
Snowscan {JQERBKV1CI3GVKNSE3Q7RGE}
Optimism {66NSFRNV1ZD4I8757MAHCJVXF}
Arbiscan {SVGA7K3yBJEQ69AFKJF4VHV}

**Comment by FKNCOIN** [comment]
K

**Pull request merged**